### ISSUE TITLE
Ubuntu AMI with unattended-upgrades package

### ISSUE THREAD

**Issue by flfernandez** [sky-uk/clusterverse#30]
Ubuntu 16.04 public AMIs come with the package "unattended-upgrades" installed and enabled by default. This causes errors while bootstrapping instances and trying to install packages due to the dpkg lock.

```
TASK [clusterverse/config : Filebeat | Download and install filebeat from elastic.co] 
******************************
FAILED - RETRYING: Filebeat | Download and install filebeat from elastic.co (5 retries left).
...
FAILED - RETRYING: Filebeat | Download and install filebeat from elastic.co (1 retries left).
fatal: [mycluster-dev-sys-a0-1582285757]: FAILED! => {"attempts": 5, "changed": false, "msg": "dpkg --force-confdef --force-confold -i /root/.ansible/tmp/ansible-moduletmp-1582285964.9641545-gns73yvp/filebeat-7.3.1-amd64e4w6h83b.deb failed", "stderr": "dpkg: error: dpkg frontend is locked by another process\n", "stderr_lines": ["dpkg: error: dpkg frontend is locked by another process"], "stdout": "", "stdout_lines": []}
fatal: [mycluster-dev-sys-b0-1582285757]: FAILED! => {"attempts": 5, "changed": false, "msg": "dpkg --force-confdef --force-confold -i /root/.ansible/tmp/ansible-moduletmp-1582285965.9946988-vutyt_nx/filebeat-7.3.1-amd649t5_cpin.deb failed", "stderr": "dpkg: error: dpkg frontend is locked by another process\n", "stderr_lines": ["dpkg: error: dpkg frontend is locked by another process"], "stdout": "", "stdout_lines": []}
fatal: [mycluster-dev-sys-c0-1582285757]: FAILED! => {"attempts": 5, "changed": false, "msg": "dpkg --force-confdef --force-confold -i /root/.ansible/tmp/ansible-moduletmp-1582285966.1172757-i2ktma72/filebeat-7.3.1-amd64puyjhnv3.deb failed", "stderr": "dpkg: error: dpkg frontend is locked by another process\n", "stderr_lines": ["dpkg: error: dpkg frontend is locked by another process"], "stdout": "", "stdout_lines": []}
```

The playbook should check and possibly stop unattended-upgrades before attempting to install any package. For example:

```
    - name: Disable unattended upgrades for Ubuntu
      service:
        name: unattended-upgrades
        state: stopped
        enabled: no
      when: ansible_os_family == "Debian"
```

Maybe worth considering reverting to enabled afterwards so the original state of the AMI is maintained.

**Comment by dseeley** [comment]
Is this not fixed by #28 ?

**Comment by leslieonline27** [comment]
> Is this not fixed by #28 ?

Yes, but in that commit unattended upgrades are allowed to complete and then removed completely.  There is probably some discussion required around whether we want to honour the initial configuration of the AMI?  Not all AMI's are created equal and unattended upgrades might be desired, although not sure they would be appropriate in Production systems.

Still, should `clusterverse` be making these kinds of configuration changes to OS configuration without either a. letting the user decide whether they want to or b. documenting the fact explicitly
